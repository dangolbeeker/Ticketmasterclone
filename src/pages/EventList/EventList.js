// Packages
import React, { Component } from "react";

// Components
import EventTable from '../../components/EventTable/EventTable';
import Search from '../../components/Search/Search'
import Footer from '../../components/Footer/Footer'


class EventList extends Component {
    constructor(props) {
        super(props);

        document.title = "Ticket Master Events"
    }

    render() {
        return (
            <div>
                <Search />
                <EventTable />
                <Footer />
            </div>
        );
    }
}


export default EventList;
