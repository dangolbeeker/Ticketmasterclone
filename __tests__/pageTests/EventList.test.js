// Packages
import React from 'react';
import { configure, shallow } from 'enzyme';
import Adapter from 'enzyme-adapter-react-16';

// Pages
import EventList from '../../src/pages/EventList/EventList'


configure({ adapter: new Adapter() });

describe('EventList component', () => {
    it('Should render without errors', () => {
        const wrapper = shallow(<EventList />);
        expect(wrapper.exists()).toBe(true)
    });

    test('Snaphot testing', () => {
        const wrapper = shallow(
            <EventList />
        );
        expect(wrapper).toMatchSnapshot();
    })
});
