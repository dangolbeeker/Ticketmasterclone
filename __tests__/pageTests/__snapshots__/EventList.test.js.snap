// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`EventList component Snaphot testing 1`] = `ShallowWrapper {}`;
